pkgs/jnigen/test/kotlin_test/bindings/kotlin.dart

```diff
@@ -1799,7 +2281,7 @@ class Operators extends jni$_.JObject {
           jni$_.JniResult Function(jni$_.Pointer<jni$_.Void>,
               jni$_.JMethodIDPtr, jni$_.Pointer<jni$_.Void>)>();
 
-  /// from: `public final com.github.dart_lang.jnigen.Operators times(com.github.dart_lang.jnigen.Operators operators)`
+  /// from: `public operator fun times(op: Operators): Operators`
```

Comment:
OK, this is nicer to look at but could be confusing if the user has multiple classes named `Operators`. I'd rather keep the fully qualified name including the package name here which is still a valid Kotlin code.

Reply:
I think, I am leaning towards using fully qualified names everywhere. WDYT?